.changeset/yellow-dolls-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/form-field': patch
+---
+
+Removes 8px gap between icons. The gap will always be 4px.
```

Comment:
"Changes the gap between icons from 8px to 4px for all sizes."

Reply:
this is better; mine sound threatening.